authentication/auth.go

```diff
@@ -492,6 +492,16 @@ func GenerateTokenFromDefaultsV2(schemaURL string, accountServiceURL string, url
 		updatedData[key] = value
 	}
 
+	/*
```

Comment:
Makes sense, just because of the confusing layout, I thought it would make sense to have a longer explanation with context, but it may be a bit too long 😅. I've made the comment smaller 👍